zerver/management/commands/mobile_push_service.py

```diff
@@ -54,21 +60,73 @@ def handle(self, *args: Any, **options: Any) -> None:
                     "in /etc/zulip/settings.py (remove the '#')"
                 )
 
+        if options["register"]:
+            self._handle_register_subcommand(*args, **options)
+        elif options["rotate_key"]:
+            self._handle_rotate_key_subcommand(*args, **options)
+
+    def _request_push_notification_bouncer_url(self, url: str, params: Dict[str, Any]) -> Response:
+        registration_url = settings.PUSH_NOTIFICATION_BOUNCER_URL + url
+        session = PushBouncerSession()
+        try:
+            response = session.post(registration_url, params=params)
+        except requests.RequestException:
+            raise CommandError(
+                "Network error connecting to push notifications service "
+                f"({settings.PUSH_NOTIFICATION_BOUNCER_URL})",
+            )
+        try:
+            response.raise_for_status()
+        except requests.HTTPError:
+            content_dict = response.json()
+            raise CommandError("Error: " + content_dict["msg"])
+
+        return response
```

Comment:
Can you do this error-handling refactoring in preparatory commits?  Renaming this tool is a sensitive operation that will need to be merged carefully, since we don't want to make our documentation confusing, so we want only changes necessary to the renaming operation in that commit.